### PR TITLE
Add notes for using devilution-comparer.

### PR DESCRIPTION
**Pull request by mewmew**
Thanks to @AJenbo and @seritools for the help in writing this

### CODE REVIEW

File: Support/CONTRIBUTING.md [hunk 1]
```diff
@@ -138,3 +138,61 @@ release.
...
+* Step 5:
+To get a function for comparison
+Build:
+`docker run --rm -ti -v $(pwd):/root/devilution -e MAKE_BUILD=pdb diasurgical/riivaaja`
+Generate diff:
+`devilution-comparer Diablo_original.exe Diablo.exe <function_name>`
+You can add `--no-mem-disp` if you want a cleaner output but this can also hide valuable details
+This will generate a `orig.asm` and `compare.asm` that you can compare in your favorit `diff` application, in the folder that you can the command from.
+
+To use riivaaja as instead of installing Wine create wine in your `$PATH` and add this content:
```
Comment (AJenbo) [inline Support/CONTRIBUTING.md]: "riivaaja as" -> "Riivaaja"

File: Support/CONTRIBUTING.md [hunk 2]
```diff
@@ -138,3 +138,61 @@ release.
...
+Or install `devililution-comparer` from source:
+1. `git clone https://github.com/diasurgical/devililution-comparer`
+2. `cd devililution-comparer`
+3. `cargo build --release`
+4. `cp cvdump.exe target/release/`
+5. `cp comparer-config.toml target/release/`
+
+Clone Devilution nightly, build and compare against the original Diablo binary:
+1. `git clone https://github.com/diasurgical/devil-nightly`
+2. `make MAKE_BUILD=pdb -f MakefileVC`
```
Comment (AJenbo) [inline Support/CONTRIBUTING.md]: This part requires that the correct version of MS VC++ 6 is installed on the users machine. and is what Riivaaja (Docker) is an alternative for.

Comment (AJenbo) [reply Support/CONTRIBUTING.md]: Maybe we can combine "Comparing a function with the original exe" and "Using devilution-comparer with Wine" as they are basically the same except for the VC dependency

File: Support/CONTRIBUTING.md [hunk 3]
```diff
@@ -138,3 +138,63 @@ release.
...
+* Step 3:
+Get the Diablo 1.09 exe
+* Step 4:
+If not on Windows Devilution-comparer requires Wine, either install Wine or use Riivaaja as a proxy (more on this later if you would like to go this route).
+* Step 5:
+
+### To get a function for comparison
+
+Build:
+`docker run --rm -ti -v $(pwd):/root/devilution -e MAKE_BUILD=pdb diasurgical/riivaaja`
```
Comment (AJenbo) [inline Support/CONTRIBUTING.md]: the -ti isn't actually needed

**Comment by mewmew** [comment]
Thanks for the review @AJenbo, I've updated to address your comments.

**Pull request merged**